### PR TITLE
Iterative builder

### PR DESCRIPTION
**Pull request by jordansread**
is a WIP for #193

### CODE REVIEW

**Comment by jordansread** [comment]
I could use a review if others are up for it. 

I would like to have legends iteratively build too, but that code is kinda huge right now to tackle. I punted and added the legend name to each call, then ignore it in print. Kind of breaks the pattern, but it was a way around a gigantic overhaul of `legend`

File: R/print.R [hunk 1]
```diff
@@ -71,10 +71,8 @@ print.gsplot <- function(x, ...){
...
   }
 
-  draw_legend(x)
-  draw_axis(x)
-
-  # par(defaultPar)
+  draw_legend(views)
+  draw_axis(views)
   
 }
```
Comment (ldecicco-USGS) [inline R/print.R:78]: I can't seem to write a comment next to a picture...but it's interesting how the ticks are behaving now. The top ones cover the legend, the left don't

Comment (jordansread) [inline R/print.R:78]: see this line: https://github.com/USGS-R/gsplot/pull/225/files#diff-832bc7f54d2e9d5191b3bdcef9979848R20

I think that is why

Comment (jordansread) [inline R/print.R:78]: and https://github.com/USGS-R/gsplot/issues/224

**Pull request merged**